models/resourcequota.go

```diff
@@ -0,0 +1,48 @@
+package models
+
+import (
+	"fmt"
+)
+
+type ResourceQuotaSpec struct {
+	Hard CustomResourceQuota
+}
+
+type ResourceQuota struct {
+	APIVersion string `yaml:"apiVersion"`
+	Kind       string
+	Metadata   Metadata
+	Spec       ResourceQuotaSpec
+}
+
+type CustomResourceQuota struct {
+	LimitsCPU       string `yaml:"limits.cpu,omitempty"`
+	LimitsMemory    string `yaml:"limits.memory,omitempty"`
+	LimitsStorage   string `yaml:"limits.storage,omitempty"`
```

Comment:
I don't think there's such a thing as `limits.storage`

Reply:
great catch, I have updated this.